app/Makefile

```diff
@@ -26,6 +26,8 @@ include $(CURDIR)/../deps/ledger-zxlib/makefiles/Makefile.installer_script
 
 include $(BOLOS_SDK)/Makefile.defines
 
+DEFINES += HAVE_SWAP
```

Comment:
I think that we don't need to deliver swap feature yet

Reply:
Ledger asked for it👌🏻